src/simplewallet/simplewallet.cpp

```diff
@@ -5114,27 +5116,23 @@ bool simple_wallet::check_daemon_rpc_prices(const std::string &daemon_url, uint3
 {
   try
   {
-    const auto nodes = m_wallet->get_public_nodes(false);
-    for (const auto &node: nodes)
+    auto i = m_claimed_cph.find(daemon_url);
```

Comment:
Isn't `m_claimed_cph` going to be empty here if `public_nodes` was not called prior to calling `set_daemon`?

Reply:
It will be, yes.

Reply:
In that case, there's no check, since you have nothing to check with.